### PR TITLE
Fork time chart

### PR DESCRIPTION
**Pull request by 7091lapS**
- Wire up new fork time metric
- Refactor csv
screen shot 2017-09-25 at 14 57 01

### CODE REVIEW

**Comment by 7091lapS** [comment]
[Graph Fork Time](https://trello.com/c/kBBqcCPC/106-graph-fork-time)

File: public/js/components/Charts/AreaChartWrap.js [hunk 1]
```diff
@@ -1,56 +1,37 @@
...
+            theme={customisedTheme}
+            series={isStacked ? data.percent : data.absolute}
+            xAxis={{
+                label: xLabel,
+                scale
+            }}
+            yAxis={{
+                label: isStacked ? yLabelStacked : yLabel,
+                scale: 'linear',
+                tickFormat: (d) => isStacked ? `${d}%` : d
```
Comment (susiecoleman) [inline public/js/components/Charts/AreaChartWrap.js]: What is `d`?

File: public/js/reducers/chartsReducer.js [hunk 2]
```diff
@@ -97,6 +100,34 @@ const charts = (state = initialState, action) => {
...
+        const forkTimeSeries = [{ data: forkTimeData }];
+        const seriesWithLabels = forkTimeSeries.map(series => {
+            return {
+                data: series.data.map((dataPoint) => {
+                    const date = moment(dataPoint['issueDate']).utc();
+                    return {
+                        x: date.valueOf(),
+                        y: dataPoint.secondsUntilFork,
+                        label: `${(dataPoint.secondsUntilFork / 3600).toFixed(1)} hours`,
+                        size: 2.5
```
Comment (susiecoleman) [inline public/js/reducers/chartsReducer.js]: Why 2.5?

File: public/js/components/Charts/Charts.js [hunk 3]
```diff
@@ -56,8 +66,25 @@ const Charts = ({ charts, isUpdating, toggleStackChart }) => {
...
+                        error={charts.forkTime.error}
+                        chartType={'ForkTime'}
+                        getClassName={getClassName}
+                        filterVals={filterVals}
+                        titleHeader={renderTitle('forkTime')}
+                        hasCsvButton={true}
+                        scale='time'
+                        data={charts.forkTime.data}
+                        height={250}
+                        yLabel='Time since forkage'
```
Comment (susiecoleman) [inline public/js/components/Charts/Charts.js]: This should probably be `Time since fork` I don't think editorial will be familiar with `forkage`

Comment (7091lapS) [reply public/js/components/Charts/Charts.js]: haha this is just my bad english

File: public/js/reducers/chartsReducer.js (hunk 2 above)
Comment (7091lapS) [reply public/js/reducers/chartsReducer.js]: this is the radius of a single dot: I set it to 2.5 which makes dots smaller than the default value, but still big enough

File: public/js/components/Charts/AreaChartWrap.js (hunk 1 above)
Comment (7091lapS) [reply public/js/components/Charts/AreaChartWrap.js]: datum!

**Comment by susiecoleman** [comment]
Thanks for the extra info. I think changing forkage to fork would be good but apart from that I'd say this is good to go 🎉

**Pull request merged**